bobtemplates/odoo/hooks.py

```diff
@@ -187,8 +187,15 @@ def post_render_readme(configurator, addon_name=""):
         if isinstance(variables["addon.oca"], bool)
         else to_boolean(False, False, variables["addon.oca"])
     )
+    generate_readme = variables.get("addon.readme", False)
```

Comment:
```suggestion
    generate_readme = variables.get("addon.readme", True)
```
Don't kill me :smile:

Reply:
Oops